### DISCUSSION TITLE
Fetch Data Based on a Dynamic Link Field

### DISCUSSION THREAD

**Discussion by Mr-NumberOne** [Sana-a-Uni/Academia#16] [Q&A]
Hey, there everyone 

I had a question,

 I wanted to configure a virual field based on a dynamic field. However, I couldn't do that by setting the "fetch from" from within the docype, since it is 
based on a dynamic link field not a link field the targeted doctype and field to be fetched couldn't be set

So if anyone has an idea on how this could be implement that would be greatly appreciated.🌹
@BakrAldubai

**Comment by BakrAldubai** [comment]
@Ali-Alnosairi  write the solution here, please
